packages/core/EventManager.ts

```diff
@@ -102,12 +102,18 @@ export default class EventManager {
     results.push(...(await this.createAllCalendarEvents(evt)));
 
     const referencesToCreate = results.map((result) => {
+      const createdEventObj = JSON.parse(result?.createdEvent as string);
       return {
         type: result.type,
-        uid: result.createdEvent?.id?.toString() ?? "",
-        meetingId: result.createdEvent?.id?.toString(),
-        meetingPassword: result.createdEvent?.password,
-        meetingUrl: result.createdEvent?.url,
+        uid:
+          result.type === "office365_calendar"
```

Comment:
Could we create a variable for "office365_calendar" so if needed we only have to change in one place?

Reply:
@dallenpyrah  what exactly do you mean by creating a new variable ? do you mean like storing "office365_calendar" in one variable and then use it to compare?

Reply:
We should already have a constant somewhere

Reply:
This code is outdated